src/main/java/org/apache/commons/imaging/palette/ColorSpaceSubset.java

```diff
@@ -49,11 +49,11 @@
         rgb = -1;
     }
 
-    ColorSpaceSubset(final int total, final int precision, final int[] mins, final int[] maxs) {
+    ColorSpaceSubset(final int total, final int precision, final int[] min, final int[] max) {
         this.total = total;
         this.precision = precision;
-        this.mins = mins;
-        this.maxs = maxs;
+        this.mins = min;
+        this.maxs = max;
```

Comment:
mins & maxs ☝️

Reply:
mmm, true.

Reply:
Thanks for fixing these so quickly @arturobernalg ! I will have to merge it tomorrow morning as I'm in a different notebook 👍

Reply:
thanks men